include/openssl/pem.h

```diff
@@ -88,12 +88,8 @@ extern "C" {
 #define PEM_STRING_PUBLIC "PUBLIC KEY"
 #define PEM_STRING_RSA "RSA PRIVATE KEY"
 #define PEM_STRING_RSA_PUBLIC "RSA PUBLIC KEY"
-
-#ifdef ENABLE_DILITHIUM
 #define PEM_STRING_DILITHIUM3 "DILITHIUM3 PRIVATE KEY"
 #define PEM_STRING_DILITHIUM3_PUBLIC "DILITHIUM3 PUBLIC KEY"
```

Comment:
can we remove these? they're [not used anywhere](https://github.com/search?q=repo%3Aaws%2Faws-lc%20PEM_STRING_DILITHIUM3_PUBLIC&type=code).

Reply:
removed in [3c9fde8](https://github.com/aws/aws-lc/pull/2082/commits/3c9fde88fcf6ba11630b567249245199e33f3d71)